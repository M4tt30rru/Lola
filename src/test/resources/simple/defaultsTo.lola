##Import "std.lola"
##Find ##Expression(_exp1) ?? ##Expression(_exp2)
 ##replace Optional.of(##(_exp1)).orElse(##(_exp2))

f() ?? d1 ?? d2
