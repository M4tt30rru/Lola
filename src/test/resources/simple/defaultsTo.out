Optional.of(Optional.of(f()).orElse(d1)).orElse(d2)
